packages/ndla-ui/src/Layout/LayoutItem.tsx

```diff
@@ -6,36 +6,28 @@
  *
  */
 
-import { HTMLAttributes, ReactNode } from "react";
+import { HTMLAttributes, ReactNode, useMemo } from "react";
 
 interface Props extends HTMLAttributes<HTMLElement> {
   children?: ReactNode;
-  layout?: string;
+  layout?: "extend" | "center";
 }
 
 export const LayoutItem = ({ children, layout, ...rest }: Props) => {
-  switch (layout) {
-    case "extend": {
-      return (
-        <section className="u-10/12@desktop u-push-1/12@desktop u-10/12@tablet u-push-1/12@tablet">{children}</section>
-      );
+  const className = useMemo(() => {
+    if (layout === "extend") {
+      return "u-10/12@desktop u-push-1/12@desktop u-10/12@tablet u-push-1/12@tablet";
+    } else if (layout === "center") {
+      return "u-7/10@desktop u-push-3/20@desktop u-10/12@tablet u-push-1/12@tablet";
     }
-    case "center": {
-      return (
-        <section className="u-4/6@desktop u-push-1/6@desktop u-10/12@tablet u-push-1/12@tablet">{children}</section>
-      );
-    }
-    case "full": {
-      return <section className="u-1/1@desktop">{children}</section>;
-    }
-    default: {
-      return (
-        <section className="o-layout__item" {...rest}>
```

Comment:
Kanskje noe her som kan tas vekk også?

Reply:
Ja, det har jeg da!

Reply:
Du har fjerna scss for o-layout__item tidligere?

Reply:
Åja, nei. Det er inuit-greier. Fjernet det fordi vi aldri bruker default-casen. Da kan det like så godt være en "vanlig" section.